deploy/docker/Dockerfile.alpine

```diff
@@ -1,6 +1,7 @@
 ARG BUILD_FROM=ghcr.io/emqx/emqx-builder/5.0-26:1.13.4-24.3.4.2-1-alpine3.15.1
 ARG RUN_FROM=alpine:3.15.1
-FROM ${BUILD_FROM} AS builder
+ARG BUILDPLATFORM=linux/amd64
```

Comment:
why this is needed?

Reply:
to pick correct platform from multiplatform image, I've seen some issues when building on arm64 platform